test/parallel/test-fs-rm.js

```diff
@@ -280,3 +282,44 @@ function removeAsync(dir) {
     message: /^The value of "options\.maxRetries" is out of range\./
   });
 }
+
+{
+  if (!common.isIBMi && (common.isWindows || process.getuid() !== 0)) {
```

Comment:
I'm guessing this is there to stop this test from running on IBMi and Android. Could you please explain why we are doing this?

Reply:
That's what other 'EPERM' and 'EACCESS' tests do

Reply:
Ah, to prevent it from being run with root privileges. Yes, I think it would be more clear if we added in that comment.